src/misc/lv_types.h

```diff
@@ -79,6 +79,8 @@ typedef uint32_t lv_uintptr_t;
 
 #if defined(PYCPARSER)
 #define LV_FORMAT_ATTRIBUTE(fmtstr, vararg)
+#elif defined(__CC_ARM)
+#define LV_FORMAT_ATTRIBUTE(fmtstr, vararg)
```

Comment:
Unless I missed something, this should be more concise.

```suggestion
#if defined(PYCPARSER) || defined(__CC_ARM)
#define LV_FORMAT_ATTRIBUTE(fmtstr, vararg)
```